### PR TITLE
Updates based on Cromwell 62 release: BT-275 BT-36 BT-274

### PR DESCRIPTION
**Pull request by kshakir**
- To the changelog:
  - Added entry for already released 62
  - Added placeholder for next release 63
- For homebrew:
  - Remove extra slash added to generated URLs
  - Changed default publishing instructions to include homebrew
  - Added validation of brew style according to guidelines
  - Fixed casing of 'cromwell' in PR name
- For the publishing GitHub token scopes:
  - updated instructions
  - updated validation
  - gracefully error with helpful messages
  - added an example image
- Removed attempt to publish from dbms tests

### CODE REVIEW

**Comment by kshakir** [comment]
For BT-36's explicit A/C, and though not requested, also demo BT-275's fix see https://github.com/kshakir/homebrew-core/pull/1/files Note for future lurkers: will be deleting that entire repo after this PR merges!

File: publish/publish_workflow.wdl [hunk 1]
```diff
@@ -237,10 +237,10 @@ task prepGithub {
             --header "Authorization: token $(cat ~{githubTokenFile})" \
             --head \
             https://api.github.com/rate_limit \
-        | grep -i -F 'X-OAuth-Scopes:' > scopesHeader.txt
+        | grep -i -F 'X-OAuth-Scopes:' > scopesHeader.txt || true
```
Comment (cjllanwarne) [inline publish/publish_workflow.wdl:240]: This is "similar but different" from what I removed in https://github.com/broadinstitute/cromwell/pull/6315. Assuming you tested this, 👍  and thanks

Comment (kshakir) [reply publish/publish_workflow.wdl:240]: > Assuming you tested this

Yep. Ran as part of the generated PR link in the comment above. That ` || true` is the difference between "wtf" and "oooooh". There are proper error messages generated below if that doesn't exit the script right there.

**Review by mcovarr — APPROVED** [review]

File: publish/publish_workflow.wdl [hunk 2]
```diff
@@ -237,10 +237,10 @@ task prepGithub {
...
 
-        # All of these scopes are required
-        required_scopes=(repo user)
+        # All of these scopes are required with an exact match
```
[the remark this answers is not in the document]
Comment (kshakir) [reply publish/publish_workflow.wdl:242]: > maybe this at least means that something like a blanked user scope would be rejected

Yes, that's what the message is trying to convey. `user` is not allowed but `user:email` is allowed. Going to merge as is assuming Travis is happy and let folks PR this more in the future (GitHub keeps refining scopes for security purposes. I feel like we'll be back here soon.)

**Pull request merged**